swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/Cacheable.kt

```diff
@@ -58,3 +62,27 @@ open class InMemoryCache<out T : Cacheable>(
 
   val log: Logger = LoggerFactory.getLogger(javaClass)
 }
+
+open class InMemorySingletonCache<out T : Cacheable>(
```

Comment:
Not at all necessary, but Cacheable inherits from Named which adds the name field requirement (though it is nullable.) Named resources make sense when always caching a set of resources, but it doesn't make sense for the singleton case. It'd be nice to remove the name="default" designation from the singleton caches.

Reply:
I'll do that later - want to properly think through a refactor.